### PR TITLE
Rands AFD patches

### PR DESCRIPTION
**Pull request by ACakeWearingAHat**
some minor hardcodes to account for abilities and items

### CODE REVIEW

File: data/random-teams.ts [hunk 1]
```diff
@@ -1219,11 +1219,13 @@ export class RandomTeams {
 		case 'Moxie':
 			return (counter.Physical < 2 || hasMove['stealthrock'] || hasMove['defog']);
 		case 'Neutralizing Gas':
-			return !hasMove['toxicspikes'];
+			return (!hasMove['toxicspikes'] && species.id !== 'miasmaw');
```
Comment (AnnikaCodes) [inline data/random-teams.ts:1222]:
```suggestion
			return !hasMove['toxicspikes'] && species.id !== 'miasmaw';
```

File: data/random-teams.ts [hunk 2]
```diff
@@ -1990,7 +1992,7 @@ export class RandomTeams {
 			if (!species.exists) continue;
 
 			// AFD enforcement of at least 1 CAP pokemon per team
-			if (!isMonotype && pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
+			if (pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
```
Comment (AnnikaCodes) [inline data/random-teams.ts:1995]:
```suggestion
			if (!isMonotype && pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
```
this is necessary, there arent enough CAPs to make monotype teams with them enforced

Comment (ACakeWearingAHat) [reply data/random-teams.ts]: i didn't do this on purpose, this is just a conflict with your other thing

File: data/random-teams.ts [hunk 3]
```diff
@@ -1339,7 +1341,7 @@ export class RandomTeams {
 		isDoubles: boolean
 	) {
 		// not undefined — we want "no item" not "go find a different item"
-		if (hasMove['acrobatics'] && ability !== 'Ripen') return ability === 'Grassy Surge' ? 'Grassy Seed' : '';
+		if (hasMove['acrobatics'] && ability !== 'Ripen' && species.id !== 'cawmodore') return ability === 'Grassy Surge' ? 'Grassy Seed' : '';
```
Comment (AnnikaCodes) [inline data/random-teams.ts:1344]:
```suggestion
		if (hasMove['acrobatics'] && ability !== 'Ripen' && species.id !== 'cawmodore') {
			return ability === 'Grassy Surge' ? 'Grassy Seed' : '';
		}
```

**Comment by AnnikaCodes** [comment]
Thanks

**Pull request merged**